index.js

```diff
@@ -127,8 +154,9 @@ NodePreGypGithub.prototype.publish = function(options) {
 		if(err) {console.error(err); return;}
 		
 		release	= (function(){ // create a new array containing only those who have a matching version.
+			var tag_name = options.tag_name || this.package_json.version;
 			data = data.filter(function(element, index, array){
-				return element.tag_name === this.package_json.version;
+				return element.tag_name === tag_name;
```

Comment:
On [line 63](https://github.com/bchr02/node-pre-gyp-github/pull/11/files#diff-168726dbe96b3ce427e7fedce31bb0bcR63) options.tag_name is defaulted to this.package_json.version and the default is overridden [(here)](https://github.com/bchr02/node-pre-gyp-github/pull/11/files#diff-168726dbe96b3ce427e7fedce31bb0bcR71) if someone uses the --tag option. Therefore, we can change this line to: ```return element.tag_name === options.tag_name;```